ngrinder-core/src/main/java/net/grinder/engine/agent/PropertyBuilder.java

```diff
@@ -205,7 +204,7 @@ public String buildJVMArgumentWithoutMemory() {
 			addAdditionalJavaOpt(jvmArguments);
 		}
 
-		String jdkVersion = getJDKVersion();
+		String jdkVersion = System.getProperty("java.specification.version");

```

Comment:
`getJDKVersion()` method called only here. And it is not related with Oshi unlike the other methods. So, remove the method and change to check system property here directly.

Reply:
This change also make Script Validation function doesn't depends on SystemInfoUtils. So, I expect that NoClassDefFoundError does not occur anymore.